monitor/services/src/main/java/org/datacleaner/monitor/server/components/ComponentHandler.java

```diff
@@ -74,43 +76,48 @@
 
     public static ObjectMapper mapper = Serializator.getJacksonObjectMapper();
 
-    private final String componentName;
-    private DataCleanerConfiguration dcConfiguration;
+    private final String _componentName;
+    private final DataCleanerConfiguration _dcConfiguration;
+    private final StringConverter _stringConverter;
```

Comment:
Please note, that simply reusing the StringConverter inside ComponentHandler does not solve problem of performance, if the mehod "deserialize" of StringConverter will be not changed too. Because current performance problem is that the "deserialize" method creates and initializes a DelegateConverter every time again.